packages/table/src/Table.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import { Story, Meta } from '@storybook/react'
+import LdTable from './LdTable'
```

Comment:
Can we lose the `Ld` prefix? The npm prefix already has `@lidofinance` in it, so I don't think we need it here. `Table`, `Thead`, `Tbody` sounds better.